tests/ctst/features/iam-policies/AssumeRoleWithWebIdentity.feature

```diff
@@ -0,0 +1,192 @@
+      | GetBucketVersioning             | STORAGE_MANAGER       | with           | does not exist |
+      | GetBucketAcl                    | STORAGE_MANAGER       | without        | does not exist |
+      | ListObjectsV2                   | STORAGE_MANAGER       | without        | exists         |
+      | ListObjectVersions              | STORAGE_MANAGER       | with           | exists         |
+      | DeleteObjects                   | STORAGE_MANAGER       | without        | exists         |
+      | HeadObject                      | STORAGE_MANAGER       | without        | exists         |
+      | CopyObject                      | STORAGE_MANAGER       | without        | exists         |
+      | GetObjectTagging                | STORAGE_MANAGER       | without        | exists         |
+      | GetObjectTagging                | STORAGE_MANAGER       | with           | exists         |
+      | PutObjectTagging                | STORAGE_MANAGER       | without        | exists         |
+      | PutBucketLifecycleConfiguration | STORAGE_MANAGER       | without        | does not exist |
+      | GetObject                       | STORAGE_MANAGER       | with           | exists         |
+      | GetObjectTagging                | STORAGE_MANAGER       | with           | exists         |
+      | DeleteObjectTagging             | STORAGE_MANAGER       | with           | exists         |
+      | DeleteObjectTagging             | STORAGE_MANAGER       | without        | exists         |
+      | PutObjectTagging                | STORAGE_MANAGER       | without        | exists         |
+      | PutObjectTagging                | STORAGE_MANAGER       | with           | exists         |
+      | GetObjectAcl                    | STORAGE_MANAGER       | with           | exists         |
+      | GetObjectAcl                    | STORAGE_MANAGER       | without        | exists         |
+      | PutObjectAcl                    | STORAGE_MANAGER       | with           | exists         |
+      | PutObjectAcl                    | STORAGE_MANAGER       | without        | exists         |
+      | PutBucketTagging                | STORAGE_MANAGER       | without        | does not exist |
+      | DeleteBucketTagging             | STORAGE_MANAGER       | without        | does not exist |
+      | PutBucketReplication            | STORAGE_MANAGER       | with           | does not exist |
+      | MetadataSearch                  | STORAGE_ACCOUNT_OWNER | without        | does not exist |
+      | PutObject                       | STORAGE_ACCOUNT_OWNER | without        | does not exist |
+      | PutObjectAcl                    | STORAGE_ACCOUNT_OWNER | without        | does not exist |
+      | GetObject                       | STORAGE_ACCOUNT_OWNER | without        | exists         |
+      | GetObject                       | STORAGE_MANAGER       | with           | exists         |
```

Comment:
This line is duplicated; I can already see it at line 24 and 39 (and here, 56)

Might be worth checking if other lines are not affected?

Reply:
This was due to a set of lines we were copy pasting to add versioned/non-versioned cases for the same APIs, I thought I was being careful but I missed this, good catch, thanks

Reply:
892d1348d0be9ca19bd3682978ec006da01430a4